Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+from Orange.util import scale
+from Orange.misc import DistMatrix
+from Orange.widgets import widget, gui, settings
+
+
+class OWDistanceTransformation(widget.OWWidget):
+    name = "Distance Transformation"
+    description = "Transform distances according to selected criteria."
+    icon = "icons/DistancesTransformation.svg"
+
+    inputs = [("Distances", DistMatrix, "set_data")]
+    outputs = [("Distances", DistMatrix)]
+
+    want_main_area = False
+
+    normalization_method = settings.Setting(0)
+    inversion_method = settings.Setting(0)
+
+    normalization_options = (
+        ("No normalization", lambda x: x),
+        ("To interval [0, 1]", lambda x: scale(x, min=0, max=1)),
+        ("Sigmoid function: 1/(1+exp(-x))", lambda x: 1/(1+np.exp(-x))),
+    )
+
+    inversion_options = (
+        ("No inversion", lambda x: x),
+        ("-X", lambda x: -x),
+        ("1 - X", lambda x: 1-x),
+        ("Max - X", lambda x: np.max(x) - x),
```

Comment:
Max of what, rows, cols, matrix?? Since you're already using function notation above, I'd go with `max(X)`.